<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionControlDetails</class>
 <widget class="QDialog" name="MotionControlDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion Control Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="daemonRespondingLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>MotionControlDaemon</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="statusFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="rotLabel">
        <property name="text">
         <string>Rotation
Drive</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="tiltLabel">
        <property name="text">
         <string>Tilt
Drive</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Status&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="respondingLabel">
        <property name="text">
         <string>Responding</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="rotRespondingIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="tiltRespondingIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="blankLabel1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="initializedLabel">
        <property name="text">
         <string>Initialized</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="rotMotorInitializedIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="tiltMotorInitializedIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="homedLabel">
        <property name="text">
         <string>Homed</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="rotMotorHomedIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="tiltMotorHomedIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="blankLabel3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="motorOnlabel">
        <property name="text">
         <string>Motor On</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="rotMotorOnIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="tiltMotorOnIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="unitModeLabel">
        <property name="text">
         <string>Unit Mode</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="rotUnitModeLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="tiltUnitModeLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="homingInProgressLabel">
        <property name="text">
         <string>Homing In Progress</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="rotMotorHomingIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="tiltMotorHomingIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="programRunningLabel">
        <property name="text">
         <string>Program Running</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="rotProgramRunningIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="tiltProgramRunningIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="motionStatusLabel">
        <property name="text">
         <string>Motion Status</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLabel" name="rotMotionStatusLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="tiltMotionStatusLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Errors&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="problemLabel">
        <property name="text">
         <string>Servo Problem</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLabel" name="rotProblemIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="16" column="2">
       <widget class="QLabel" name="tiltProblemIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="problemCodeLabel">
        <property name="text">
         <string>Problem Code</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLabel" name="rotProblemCodeLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="17" column="2">
       <widget class="QLabel" name="tiltProblemCodeLabel">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="motorFailureLabel">
        <property name="text">
         <string>Motor Failure</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QLabel" name="rotMotorFailureIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="18" column="2">
       <widget class="QLabel" name="tiltMotorFailureIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="cpuErrorLabel">
        <property name="text">
         <string>CPU Error</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QLabel" name="rotCpuErrorIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="19" column="2">
       <widget class="QLabel" name="tiltCpuErrorIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="stoppedByLimitLabel">
        <property name="text">
         <string>Stopped By Limit</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QLabel" name="rotStoppedByLimitIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="20" column="2">
       <widget class="QLabel" name="tiltStoppedByLimitIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLabel" name="programErrorLabel">
        <property name="text">
         <string>User Program Error</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1">
       <widget class="QLabel" name="rotProgramErrorIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="21" column="2">
       <widget class="QLabel" name="tiltProgramErrorIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="externalInhibitLabel">
        <property name="text">
         <string>External Inhibit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="rotInhibitIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="tiltInhibitIcon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="HcrGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="blankLabel2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="HcrGuiMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MotionControlDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MotionControlDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
